.github/workflows/linux-checks.yaml

```diff
@@ -166,36 +169,39 @@ jobs:
           # For more details, see:
           # https://github.com/actions/checkout/issues/299.
           ref: ${{ github.event.pull_request.head.sha }}
+      - name: Preparation - github token for gitops access
+        run: git config --global url."https://${{secrets.INS_SERVICE_PAT}}@github.com/".insteadOf "https://github.com/"
+      - name: Preparation - set prerequisite variables, full sha is ${{github.sha}}
+        env:
+          TARGET: ${{matrix.target}}
+        run: |
+          echo "::set-env name=INSOLAR_TAG::${GITHUB_SHA:0:8}"
+          echo "::set-env name=INSOLAR_NETWORK_SIZE::${TARGET}"
+          echo "::set-env name=INSOLAR_NAMESPACE::${GITHUB_SHA:0:8}-${TARGET}"
+          echo "::set-env name=INSOLAR_DESIRED_UPTIME::300"
+          echo "::set-env name=INSOLAR_BRANCH::$(echo "pull-$(echo ${{github.ref}} | cut -d '/' -f3)")"
+          echo "::set-env name=INSOLAR_NETWORK_SIZE_INT::$(echo $TARGET | awk '{gsub(/[^[0-9]]*/,"");print}')"
```

Comment:
Do we need network size here?

Reply:
Not exactly a size here, but can be used like one, later on